playbooks/roles/scorm/tasks/scorm_base.yml

```diff
@@ -0,0 +1,29 @@
+
+- name: FIXME create application user (copied over from edxapp/tasks/main.yml)
```

Comment:
It's not clear what the `FIXME` suggests. Could you please either remove it or add more details.

Reply:
I updated with a better comment.